src/content/tutorial/react-step-1/react-step-1.mdx

```diff
@@ -1,136 +1,169 @@
 ---
-title: React Step 1 
+title: Installing Carbon
 internal: false
 ---
 
 <AnchorLinks>
 
-- [Create Project](#create-project)
-- [Step 1 Branch](#step-1-branch)
-- [Installing Dependencies](#Installing-dependencies)
-- [Setting Up Sass](#setting-up-sass)
-- [Adding the UI Shell](#adding-the-ui-shell)
+- [Fork, clone & branch](#fork-clone--branch)
+- [Build & test app](#build--test-app)
+- [Install Carbon](#install-carbon)
+- [Install & build Sass](#install--build-sass)
+- [Add UI Shell](#add-ui-shell)
+- [Create pages](#create-pages)
 - [Add routing](#add-routing)
-- [Creating Our Pages](#creating-our-pages)
-- [Rendering Content With react-router-dom](#rendering-content-with-react-router-dom)
+- [Submit pull request](#submit-pull-request)
 
 </AnchorLinks>
 
-## Create project
+## Introduction
 
-[Create React App](https://github.com/facebook/create-react-app) allows you to set up a modern web application by running one command. For more information on Create React App, see its [philosophy](https://github.com/facebook/create-react-app#philosophy) and [what's included](https://github.com/facebook/create-react-app#whats-included).
+This step starts with an existing React app created with [create-react-app](https://github.com/facebook/create-react-app), installs Carbon and begins to use Carbon components. By the end of this step you will have a React app that uses the UI Shell to navigate between pages.
 
-### Set up cloned repo
-The first step is to fork and clone the tutorial repo into your own Github.
+## Fork, clone & branch
 
-#### Fork
-Fork the carbon tutorial [repo](https://github.com/carbon-design-system/carbon-tutorial).
+This tutorial has an accompanying GitHub responsitory called [carbon-tutorial](https://github.com/carbon-design-system/carbon-tutorial) that we'll use as a starting point for each step along the way.
 
-#### Clone
-From your forked repo, copy the SSH key or HTTPS url and in your terminal run
-```git clone [your fork SSH/HTTPS]```
+### Fork
 
+To begin, fork [carbon-tutorial](https://github.com/carbon-design-system/carbon-tutorial) using your GitHub account.
 
-### Set up upstream
-Set up the original repo as an upstream by running the following in your terminal:
+### Clone
+
+From your forked repository, copy the SSH key or URL and in your terminal run the two commands to get the repositiory in your local filesystem and enter that directory.
```

Comment:
repository is misspelled the second time: `... run the two commands to get the repositiory ...`

Reply:
Got it.